confidant_client/__init__.py

```diff
@@ -468,7 +482,7 @@ def create_blind_credential(
             'enabled': enabled
         }
         if store_keys:
-            data['credential_keys'] = credential_pairs.keys()
+            data['credential_keys'] = list(credential_pairs.keys())
```

Comment:
or [`tuple(...)`](https://github.com/asottile/scratch/wiki/Prefer-tuples-to-lists) shrugs